server/tournaments/index.ts

```diff
@@ -1524,6 +1526,22 @@ const tourCommands: {basic: TourCommands, creation: TourCommands, moderation: To
 				return this.sendReply(`Usage: ${cmd} <allow|disallow>`);
 			}
 		},
+		forcepublicreplays(tournament, user, params, cmd) {
+			const option = params.length ? params[0].toLowerCase() : 'on';
```

Comment:
```suggestion
			const option = params[0] || 'on';
```

`meansYes` and `meansNo` already normalize this.